mmaction/datasets/pipelines/augmentations.py

```diff
@@ -152,41 +152,35 @@ def __repr__(self):
         return repr_str
 
 
+# Note, entity box transfroms are not added to: ThreeCrop, TenCrop,
+# MultiGroupCrop.
 @PIPELINES.register_module()
 class EntityBoxRescale:
     """Rescale the entity box and proposals according to the image shape.
 
-    Required keys are "img_shape", "scale_factor", "proposals",
-    "ann.entity_boxes", added or modified keys are "ann.entity_boxes". If
-    original "proposals" is not None, "proposals" and "scores" will be added or
-    modified.
+    Required keys are "proposals", "gt_bboxes", added or modified keys are
+    "gt_bboxes". If original "proposals" is not None, "proposals" and
+    will be added or modified.
+
+    Args:
+        scale_factor (np.ndarray): The scale factor used entity_box rescaling.
     """
 
+    def __init__(self, scale_factor):
+        self.scale_factor = scale_factor
```

Comment:
Is `scale_factor` a dataset-wise parameter?

Reply:
Nope, we will not use EntityBoxXX classes explicitly in the pipeline. They are called in the corresponding ops for images (like Flip will call EntityBoxFlip, etc.). So that the scale_factor is designated by Resize.

Reply:
But this calling style will be different from others

Reply:
Yes. If we write out these EntityBoxXX ops explicitly in the pipeline. For correctness' sake, after each Img-OP, we should add a Box-Op. I think this knowledge is not straight-forward for users. It would be better to hide them from users.